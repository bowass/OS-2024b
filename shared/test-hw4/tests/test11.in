19785
28844
11521
24082
9542
1655
14950
29355
10774
4176
1464
4626
7748
3516
26587
20162
13570
16445
22357
11819
8246
20713
29502
1448
25495
7666
21773
22141
12316
29631
9675
10272
16479
24283
10459
27529
4285
17066
26348
6343
3188
4236
6367
12701
5084
20969
25181
28992
27378
4058
10373
3085
4014
13154
9356
175
17011
4447
12048
20362
31824
21769
1105
19218
2716
6881
7676
20744
31107
1067
15608
24280
10816
30464
26241
25232
26224
15764
5283
3465
4471
6621
14677
10359
25453
29853
2160
27502
793
10931
1490
14500
13157
2439
13343
22729
3149
23211
19696
17200
10462
23335
4520
28559
32363
29270
3193
6841
15516
5289
8634
21104
27469
293
11383
31007
10774
27682
9774
19558
4399
24422
10692
3953
17953
26404
19413
5751
22033
30197
16326
27596
17322
5361
5076
22147
2958
16481
29829
24961
17357
4504
17850
18506
26029
20637
22300
29365
32250
1689
16068
29052
29955
24600
10509
29357
18250
8735
5251
28872
16438
14738
5583
14056
24860
15732
10359
12267
29549
31326
7982
12030
9768
14621
18793
23271
3430
3250
7876
6390
22347
20051
30845
16623
28003
6348
22380
11712
13635
13821
26838
28009
18683
14530
30654
12500
6298
25088
32182
8414
21891
1158
11460
27624
21348
15749
12730
24584
11779
27455
22619
12111
31944
24782
11075
18838
6918
30831
26260
11842
26120
13720
13383
23662
25914
31003
9644
20771
22209
28923
10779
20402
23872
1954
30957
27531
25362
1224
16507
13117
17839
9460
18615
17075
22297
18082
21986
30209
11385
27966
19039
18876
5010
25652
25845
5941
25660
27366
11449
15378
31352
603
10038
24292
30502
6429
12110
16892
7428
3046
16020
24640
19187
7274
25551
2965
1458
25877
3675
25686
14310
13723
7955
5296
30364
17217
4610
6507
2236
32286
3468
3745
12705
19327
7039
14637
5405
26439
13812
9102
3298
17675
22483
10569
3439
25830
32242
9179
13508
14206
25869
17091
12668
6973
5006
18659
25566
15798
10046
1122
3467
9737
2136
7835
15271
12168
32140
29102
15936
11271
15634
20866
8309
5782
22639
16121
3582
13423
26031
12329
24001
8325
9466
28931
3525
25863
23344
18569
31049
24382
3993
14236
18268
26790
19681
23956
31355
31473
6364
7130
27570
9866
4222
12603
21802
1170
11781
9409
26707
21096
11547
23023
16795
11066
28117
15689
8585
27359
15528
4721
32233
19679
31398
20740
20612
27688
13766
27159
23816
12497
8483
28609
16079
6695
27768
2430
14492
25846
3111
248
30281
14958
8627
31123
3057
2591
11819
19822
28141
16843
11284
1893
5554
6731
17349
8802
30642
14420
28252
18297
28805
21001
5750
30711
27950
30854
2509
17008
24332
26607
19254
11447
31699
28630
31324
9945
19594
31611
26177
4894
2376
24040
24072
9647
15712
15413
11511
3497
8034
19514
1325
13164
7235
14757
7941
9293
4281
6434
6019
1791
26773
16260
10841
10916
23412
26045
29578
4762
25979
18382
25801
14790
22546
8198
4625
9884
659
20481
1612
12394
6638
25344
6601
22136
12694
6228
823
30030
21915
24523
2608
9434
8731
12482
11927
916
6236
17129
31349
12848