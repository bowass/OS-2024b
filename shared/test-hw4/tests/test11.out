Virtual address: 19785 Physical address: 73 Value: 0
Virtual address: 28844 Physical address: 428 Value: 0
Virtual address: 11521 Physical address: 513 Value: 0
Virtual address: 24082 Physical address: 786 Value: 23
Virtual address: 9542 Physical address: 1094 Value: 9
Virtual address: 1655 Physical address: 1399 Value: -99
Virtual address: 14950 Physical address: 1638 Value: 14
Virtual address: 29355 Physical address: 1963 Value: -86
Virtual address: 10774 Physical address: 2070 Value: 10
Virtual address: 4176 Physical address: 2384 Value: 0
Virtual address: 1464 Physical address: 2744 Value: 0
Virtual address: 4626 Physical address: 2834 Value: 4
Virtual address: 7748 Physical address: 3140 Value: 0
Virtual address: 3516 Physical address: 3516 Value: 0
Virtual address: 26587 Physical address: 3803 Value: -10
Virtual address: 20162 Physical address: 4034 Value: 19
Virtual address: 13570 Physical address: 4098 Value: 13
Virtual address: 16445 Physical address: 4413 Value: 0
Virtual address: 22357 Physical address: 4693 Value: 0
Virtual address: 11819 Physical address: 4907 Value: -118
Virtual address: 8246 Physical address: 5174 Value: 8
Virtual address: 20713 Physical address: 5609 Value: 0
Virtual address: 29502 Physical address: 5694 Value: 28
Virtual address: 1448 Physical address: 2728 Value: 0
Virtual address: 25495 Physical address: 6039 Value: -27
Virtual address: 7666 Physical address: 6386 Value: 7
Virtual address: 21773 Physical address: 6413 Value: 0
Virtual address: 22141 Physical address: 6781 Value: 0
Virtual address: 12316 Physical address: 6940 Value: 0
Virtual address: 29631 Physical address: 5823 Value: -17
Virtual address: 9675 Physical address: 1227 Value: 114
Virtual address: 10272 Physical address: 7200 Value: 0
Virtual address: 16479 Physical address: 4447 Value: 23
Virtual address: 24283 Physical address: 987 Value: -74
Virtual address: 10459 Physical address: 7387 Value: 54
Virtual address: 27529 Physical address: 7561 Value: 0
Virtual address: 4285 Physical address: 2493 Value: 0
Virtual address: 17066 Physical address: 7850 Value: 16
Virtual address: 26348 Physical address: 8172 Value: 0
Virtual address: 6343 Physical address: 8391 Value: 49
Virtual address: 3188 Physical address: 8564 Value: 0
Virtual address: 4236 Physical address: 2444 Value: 0
Virtual address: 6367 Physical address: 8415 Value: 55
Virtual address: 12701 Physical address: 8861 Value: 0
Virtual address: 5084 Physical address: 9180 Value: 0
Virtual address: 20969 Physical address: 9449 Value: 0
Virtual address: 25181 Physical address: 9565 Value: 0
Virtual address: 28992 Physical address: 9792 Value: 0
Virtual address: 27378 Physical address: 10226 Value: 26
Virtual address: 4058 Physical address: 10458 Value: 3
Virtual address: 10373 Physical address: 7301 Value: 0
Virtual address: 3085 Physical address: 8461 Value: 0
Virtual address: 4014 Physical address: 10414 Value: 3
Virtual address: 13154 Physical address: 10594 Value: 12
Virtual address: 9356 Physical address: 10892 Value: 0
Virtual address: 175 Physical address: 11183 Value: 43
Virtual address: 17011 Physical address: 7795 Value: -100
Virtual address: 4447 Physical address: 11359 Value: 87
Virtual address: 12048 Physical address: 11536 Value: 0
Virtual address: 20362 Physical address: 11914 Value: 19
Virtual address: 31824 Physical address: 12112 Value: 0
Virtual address: 21769 Physical address: 6409 Value: 0
Virtual address: 1105 Physical address: 12369 Value: 0
Virtual address: 19218 Physical address: 12562 Value: 18
Virtual address: 2716 Physical address: 12956 Value: 0
Virtual address: 6881 Physical address: 13281 Value: 0
Virtual address: 7676 Physical address: 6396 Value: 0
Virtual address: 20744 Physical address: 9224 Value: 0
Virtual address: 31107 Physical address: 13443 Value: 96
Virtual address: 1067 Physical address: 12331 Value: 10
Virtual address: 15608 Physical address: 13816 Value: 0
Virtual address: 24280 Physical address: 984 Value: 0
Virtual address: 10816 Physical address: 2112 Value: 0
Virtual address: 30464 Physical address: 13824 Value: 0
Virtual address: 26241 Physical address: 8065 Value: 0
Virtual address: 25232 Physical address: 9616 Value: 0
Virtual address: 26224 Physical address: 8048 Value: 0
Virtual address: 15764 Physical address: 14228 Value: 0
Virtual address: 5283 Physical address: 14499 Value: 40
Virtual address: 3465 Physical address: 3465 Value: 0
Virtual address: 4471 Physical address: 11383 Value: 93
Virtual address: 6621 Physical address: 14813 Value: 0
Virtual address: 14677 Physical address: 14933 Value: 0
Virtual address: 10359 Physical address: 7287 Value: 29
Virtual address: 25453 Physical address: 5997 Value: 0
Virtual address: 29853 Physical address: 15261 Value: 0
Virtual address: 2160 Physical address: 15472 Value: 0
Virtual address: 27502 Physical address: 7534 Value: 26
Virtual address: 793 Physical address: 15641 Value: 0
Virtual address: 10931 Physical address: 2227 Value: -84
Virtual address: 1490 Physical address: 2770 Value: 1
Virtual address: 14500 Physical address: 16036 Value: 0
Virtual address: 13157 Physical address: 10597 Value: 0
Virtual address: 2439 Physical address: 16263 Value: 97
Virtual address: 13343 Physical address: 16415 Value: 7
Virtual address: 22729 Physical address: 16841 Value: 0
Virtual address: 3149 Physical address: 8525 Value: 0
Virtual address: 23211 Physical address: 17067 Value: -86
Virtual address: 19696 Physical address: 17392 Value: 0
Virtual address: 17200 Physical address: 17456 Value: 0
Virtual address: 10462 Physical address: 7390 Value: 10
Virtual address: 23335 Physical address: 17703 Value: -55
Virtual address: 4520 Physical address: 11432 Value: 0
Virtual address: 28559 Physical address: 18063 Value: -29
Virtual address: 32363 Physical address: 18283 Value: -102
Virtual address: 29270 Physical address: 1878 Value: 28
Virtual address: 3193 Physical address: 8569 Value: 0
Virtual address: 6841 Physical address: 13241 Value: 0
Virtual address: 15516 Physical address: 13724 Value: 0
Virtual address: 5289 Physical address: 14505 Value: 0
Virtual address: 8634 Physical address: 18618 Value: 8
Virtual address: 21104 Physical address: 18800 Value: 0
Virtual address: 27469 Physical address: 7501 Value: 0
Virtual address: 293 Physical address: 18981 Value: 0
Virtual address: 11383 Physical address: 19319 Value: 29
Virtual address: 31007 Physical address: 13343 Value: 71
Virtual address: 10774 Physical address: 2070 Value: 10
Virtual address: 27682 Physical address: 19490 Value: 27
Virtual address: 9774 Physical address: 19758 Value: 9
Virtual address: 19558 Physical address: 17254 Value: 19
Virtual address: 4399 Physical address: 11311 Value: 75
Virtual address: 24422 Physical address: 20070 Value: 23
Virtual address: 10692 Physical address: 20420 Value: 0
Virtual address: 3953 Physical address: 10353 Value: 0
Virtual address: 17953 Physical address: 20513 Value: 0
Virtual address: 26404 Physical address: 3620 Value: 0
Virtual address: 19413 Physical address: 12757 Value: 0
Virtual address: 5751 Physical address: 20855 Value: -99
Virtual address: 22033 Physical address: 6673 Value: 0
Virtual address: 30197 Physical address: 21237 Value: 0
Virtual address: 16326 Physical address: 21446 Value: 15
Virtual address: 27596 Physical address: 7628 Value: 0
Virtual address: 17322 Physical address: 17578 Value: 16
Virtual address: 5361 Physical address: 14577 Value: 0
Virtual address: 5076 Physical address: 9172 Value: 0
Virtual address: 22147 Physical address: 6787 Value: -96
Virtual address: 2958 Physical address: 21646 Value: 2
Virtual address: 16481 Physical address: 4449 Value: 0
Virtual address: 29829 Physical address: 15237 Value: 0
Virtual address: 24961 Physical address: 21889 Value: 0
Virtual address: 17357 Physical address: 17613 Value: 0
Virtual address: 4504 Physical address: 11416 Value: 0
Virtual address: 17850 Physical address: 22202 Value: 17
Virtual address: 18506 Physical address: 22346 Value: 18
Virtual address: 26029 Physical address: 22701 Value: 0
Virtual address: 20637 Physical address: 5533 Value: 0
Virtual address: 22300 Physical address: 4636 Value: 0
Virtual address: 29365 Physical address: 1973 Value: 0
Virtual address: 32250 Physical address: 23034 Value: 31
Virtual address: 1689 Physical address: 1433 Value: 0
Virtual address: 16068 Physical address: 23236 Value: 0
Virtual address: 29052 Physical address: 9852 Value: 0
Virtual address: 29955 Physical address: 20995 Value: 64
Virtual address: 24600 Physical address: 23320 Value: 0
Virtual address: 10509 Physical address: 20237 Value: 0
Virtual address: 29357 Physical address: 1965 Value: 0
Virtual address: 18250 Physical address: 23626 Value: 17
Virtual address: 8735 Physical address: 23839 Value: -121
Virtual address: 5251 Physical address: 14467 Value: 32
Virtual address: 28872 Physical address: 456 Value: 0
Virtual address: 16438 Physical address: 4406 Value: 16
Virtual address: 14738 Physical address: 14994 Value: 14
Virtual address: 5583 Physical address: 24271 Value: 115
Virtual address: 14056 Physical address: 24552 Value: 0
Virtual address: 24860 Physical address: 21788 Value: 0
Virtual address: 15732 Physical address: 14196 Value: 0
Virtual address: 10359 Physical address: 7287 Value: 29
Virtual address: 12267 Physical address: 11755 Value: -6
Virtual address: 29549 Physical address: 5741 Value: 0
Virtual address: 31326 Physical address: 24670 Value: 30
Virtual address: 7982 Physical address: 24878 Value: 7
Virtual address: 12030 Physical address: 5118 Value: 11
Virtual address: 9768 Physical address: 19752 Value: 0
Virtual address: 14621 Physical address: 14877 Value: 0
Virtual address: 18793 Physical address: 25193 Value: 0
Virtual address: 23271 Physical address: 17127 Value: -71
Virtual address: 3430 Physical address: 3430 Value: 3
Virtual address: 3250 Physical address: 8626 Value: 3
Virtual address: 7876 Physical address: 3268 Value: 0
Virtual address: 6390 Physical address: 8438 Value: 6
Virtual address: 22347 Physical address: 4683 Value: -46
Virtual address: 20051 Physical address: 3923 Value: -108
Virtual address: 30845 Physical address: 25469 Value: 0
Virtual address: 16623 Physical address: 4591 Value: 59
Virtual address: 28003 Physical address: 25699 Value: 88
Virtual address: 6348 Physical address: 8396 Value: 0
Virtual address: 22380 Physical address: 4716 Value: 0
Virtual address: 11712 Physical address: 704 Value: 0
Virtual address: 13635 Physical address: 4163 Value: 80
Virtual address: 13821 Physical address: 4349 Value: 0
Virtual address: 26838 Physical address: 26070 Value: 26
Virtual address: 28009 Physical address: 25705 Value: 0
Virtual address: 18683 Physical address: 22523 Value: 62
Virtual address: 14530 Physical address: 16066 Value: 14
Virtual address: 30654 Physical address: 14014 Value: 29
Virtual address: 12500 Physical address: 7124 Value: 0
Virtual address: 6298 Physical address: 8346 Value: 6
Virtual address: 25088 Physical address: 9472 Value: 0
Virtual address: 32182 Physical address: 22966 Value: 31
Virtual address: 8414 Physical address: 5342 Value: 8
Virtual address: 21891 Physical address: 6531 Value: 96
Virtual address: 1158 Physical address: 12422 Value: 1
Virtual address: 11460 Physical address: 19396 Value: 0
Virtual address: 27624 Physical address: 7656 Value: 0
Virtual address: 21348 Physical address: 26212 Value: 0
Virtual address: 15749 Physical address: 14213 Value: 0
Virtual address: 12730 Physical address: 8890 Value: 12
Virtual address: 24584 Physical address: 23304 Value: 0
Virtual address: 11779 Physical address: 4867 Value: -128
Virtual address: 27455 Physical address: 7487 Value: -49
Virtual address: 22619 Physical address: 16731 Value: 22
Virtual address: 12111 Physical address: 11599 Value: -45
Virtual address: 31944 Physical address: 12232 Value: 0
Virtual address: 24782 Physical address: 23502 Value: 24
Virtual address: 11075 Physical address: 26435 Value: -48
Virtual address: 18838 Physical address: 25238 Value: 18
Virtual address: 6918 Physical address: 26630 Value: 6
Virtual address: 30831 Physical address: 25455 Value: 27
Virtual address: 26260 Physical address: 8084 Value: 0
Virtual address: 11842 Physical address: 4930 Value: 11
Virtual address: 26120 Physical address: 7944 Value: 0
Virtual address: 13720 Physical address: 4248 Value: 0
Virtual address: 13383 Physical address: 16455 Value: 17
Virtual address: 23662 Physical address: 26990 Value: 23
Virtual address: 25914 Physical address: 22586 Value: 25
Virtual address: 31003 Physical address: 13339 Value: 70
Virtual address: 9644 Physical address: 1196 Value: 0
Virtual address: 20771 Physical address: 9251 Value: 72
Virtual address: 22209 Physical address: 6849 Value: 0
Virtual address: 28923 Physical address: 507 Value: 62
Virtual address: 10779 Physical address: 2075 Value: -122
Virtual address: 20402 Physical address: 11954 Value: 19
Virtual address: 23872 Physical address: 27200 Value: 0
Virtual address: 1954 Physical address: 27554 Value: 1
Virtual address: 30957 Physical address: 25581 Value: 0
Virtual address: 27531 Physical address: 7563 Value: -30
Virtual address: 25362 Physical address: 5906 Value: 24
Virtual address: 1224 Physical address: 12488 Value: 0
Virtual address: 16507 Physical address: 4475 Value: 30
Virtual address: 13117 Physical address: 10557 Value: 0
Virtual address: 17839 Physical address: 22191 Value: 107
Virtual address: 9460 Physical address: 10996 Value: 0
Virtual address: 18615 Physical address: 22455 Value: 45
Virtual address: 17075 Physical address: 7859 Value: -84
Virtual address: 22297 Physical address: 4633 Value: 0
Virtual address: 18082 Physical address: 20642 Value: 17
Virtual address: 21986 Physical address: 6626 Value: 21
Virtual address: 30209 Physical address: 27649 Value: 0
Virtual address: 11385 Physical address: 19321 Value: 0
Virtual address: 27966 Physical address: 25662 Value: 27
Virtual address: 19039 Physical address: 27999 Value: -105
Virtual address: 18876 Physical address: 25276 Value: 0
Virtual address: 5010 Physical address: 9106 Value: 4
Virtual address: 25652 Physical address: 28212 Value: 0
Virtual address: 25845 Physical address: 28405 Value: 0
Virtual address: 5941 Physical address: 28469 Value: 0
Virtual address: 25660 Physical address: 28220 Value: 0
Virtual address: 27366 Physical address: 10214 Value: 26
Virtual address: 11449 Physical address: 19385 Value: 0
Virtual address: 15378 Physical address: 13586 Value: 15
Virtual address: 31352 Physical address: 24696 Value: 0
Virtual address: 603 Physical address: 28763 Value: -106
Virtual address: 10038 Physical address: 28982 Value: 9
Virtual address: 24292 Physical address: 996 Value: 0
Virtual address: 30502 Physical address: 13862 Value: 29
Virtual address: 6429 Physical address: 14621 Value: 0
Virtual address: 12110 Physical address: 11598 Value: 11
Virtual address: 16892 Physical address: 29436 Value: 0
Virtual address: 7428 Physical address: 6148 Value: 0
Virtual address: 3046 Physical address: 21734 Value: 2
Virtual address: 16020 Physical address: 23188 Value: 0
Virtual address: 24640 Physical address: 23360 Value: 0
Virtual address: 19187 Physical address: 28147 Value: -68
Virtual address: 7274 Physical address: 29546 Value: 7
Virtual address: 25551 Physical address: 6095 Value: -13
Virtual address: 2965 Physical address: 21653 Value: 0
Virtual address: 1458 Physical address: 2738 Value: 1
Virtual address: 25877 Physical address: 22549 Value: 0
Virtual address: 3675 Physical address: 29787 Value: -106
Virtual address: 25686 Physical address: 28246 Value: 25
Virtual address: 14310 Physical address: 30182 Value: 13
Virtual address: 13723 Physical address: 4251 Value: 102
Virtual address: 7955 Physical address: 24851 Value: -60
Virtual address: 5296 Physical address: 14512 Value: 0
Virtual address: 30364 Physical address: 27804 Value: 0
Virtual address: 17217 Physical address: 17473 Value: 0
Virtual address: 4610 Physical address: 2818 Value: 4
Virtual address: 6507 Physical address: 14699 Value: 90
Virtual address: 2236 Physical address: 15548 Value: 0
Virtual address: 32286 Physical address: 18206 Value: 31
Virtual address: 3468 Physical address: 3468 Value: 0
Virtual address: 3745 Physical address: 29857 Value: 0
Virtual address: 12705 Physical address: 8865 Value: 0
Virtual address: 19327 Physical address: 12671 Value: -33
Virtual address: 7039 Physical address: 26751 Value: -33
Virtual address: 14637 Physical address: 14893 Value: 0
Virtual address: 5405 Physical address: 24093 Value: 0
Virtual address: 26439 Physical address: 3655 Value: -47
Virtual address: 13812 Physical address: 4340 Value: 0
Virtual address: 9102 Physical address: 30350 Value: 8
Virtual address: 3298 Physical address: 8674 Value: 3
Virtual address: 17675 Physical address: 22027 Value: 66
Virtual address: 22483 Physical address: 4819 Value: -12
Virtual address: 10569 Physical address: 20297 Value: 0
Virtual address: 3439 Physical address: 3439 Value: 91
Virtual address: 25830 Physical address: 28390 Value: 25
Virtual address: 32242 Physical address: 23026 Value: 31
Virtual address: 9179 Physical address: 30427 Value: -10
Virtual address: 13508 Physical address: 16580 Value: 0
Virtual address: 14206 Physical address: 30078 Value: 13
Virtual address: 25869 Physical address: 22541 Value: 0
Virtual address: 17091 Physical address: 7875 Value: -80
Virtual address: 12668 Physical address: 8828 Value: 0
Virtual address: 6973 Physical address: 26685 Value: 0
Virtual address: 5006 Physical address: 9102 Value: 4
Virtual address: 18659 Physical address: 22499 Value: 56
Virtual address: 25566 Physical address: 6110 Value: 24
Virtual address: 15798 Physical address: 14262 Value: 15
Virtual address: 10046 Physical address: 28990 Value: 9
Virtual address: 1122 Physical address: 12386 Value: 1
Virtual address: 3467 Physical address: 3467 Value: 98
Virtual address: 9737 Physical address: 19721 Value: 0
Virtual address: 2136 Physical address: 15448 Value: 0
Virtual address: 7835 Physical address: 3227 Value: -90
Virtual address: 15271 Physical address: 30631 Value: -23
Virtual address: 12168 Physical address: 11656 Value: 0
Virtual address: 32140 Physical address: 22924 Value: 0
Virtual address: 29102 Physical address: 9902 Value: 28
Virtual address: 15936 Physical address: 23104 Value: 0
Virtual address: 11271 Physical address: 19207 Value: 1
Virtual address: 15634 Physical address: 14098 Value: 15
Virtual address: 20866 Physical address: 9346 Value: 20
Virtual address: 8309 Physical address: 5237 Value: 0
Virtual address: 5782 Physical address: 20886 Value: 5
Virtual address: 22639 Physical address: 16751 Value: 27
Virtual address: 16121 Physical address: 23289 Value: 0
Virtual address: 3582 Physical address: 3582 Value: 3
Virtual address: 13423 Physical address: 16495 Value: 27
Virtual address: 26031 Physical address: 22703 Value: 107
Virtual address: 12329 Physical address: 6953 Value: 0
Virtual address: 24001 Physical address: 27329 Value: 0
Virtual address: 8325 Physical address: 5253 Value: 0
Virtual address: 9466 Physical address: 11002 Value: 9
Virtual address: 28931 Physical address: 9731 Value: 64
Virtual address: 3525 Physical address: 3525 Value: 0
Virtual address: 25863 Physical address: 22535 Value: 65
Virtual address: 23344 Physical address: 17712 Value: 0
Virtual address: 18569 Physical address: 22409 Value: 0
Virtual address: 31049 Physical address: 13385 Value: 0
Virtual address: 24382 Physical address: 20030 Value: 23
Virtual address: 3993 Physical address: 10393 Value: 0
Virtual address: 14236 Physical address: 30108 Value: 0
Virtual address: 18268 Physical address: 23644 Value: 0
Virtual address: 26790 Physical address: 26022 Value: 26
Virtual address: 19681 Physical address: 17377 Value: 0
Virtual address: 23956 Physical address: 27284 Value: 0
Virtual address: 31355 Physical address: 24699 Value: -98
Virtual address: 31473 Physical address: 24817 Value: 0
Virtual address: 6364 Physical address: 8412 Value: 0
Virtual address: 7130 Physical address: 26842 Value: 6
Virtual address: 27570 Physical address: 7602 Value: 26
Virtual address: 9866 Physical address: 19850 Value: 9
Virtual address: 4222 Physical address: 2430 Value: 4
Virtual address: 12603 Physical address: 8763 Value: 78
Virtual address: 21802 Physical address: 6442 Value: 21
Virtual address: 1170 Physical address: 12434 Value: 1
Virtual address: 11781 Physical address: 4869 Value: 0
Virtual address: 9409 Physical address: 10945 Value: 0
Virtual address: 26707 Physical address: 25939 Value: 20
Virtual address: 21096 Physical address: 18792 Value: 0
Virtual address: 11547 Physical address: 539 Value: 70
Virtual address: 23023 Physical address: 30959 Value: 123
Virtual address: 16795 Physical address: 29339 Value: 102
Virtual address: 11066 Physical address: 26426 Value: 10
Virtual address: 28117 Physical address: 25813 Value: 0
Virtual address: 15689 Physical address: 14153 Value: 0
Virtual address: 8585 Physical address: 18569 Value: 0
Virtual address: 27359 Physical address: 10207 Value: -73
Virtual address: 15528 Physical address: 13736 Value: 0
Virtual address: 4721 Physical address: 2929 Value: 0
Virtual address: 32233 Physical address: 23017 Value: 0
Virtual address: 19679 Physical address: 17375 Value: 55
Virtual address: 31398 Physical address: 24742 Value: 30
Virtual address: 20740 Physical address: 9220 Value: 0
Virtual address: 20612 Physical address: 5508 Value: 0
Virtual address: 27688 Physical address: 19496 Value: 0
Virtual address: 13766 Physical address: 4294 Value: 13
Virtual address: 27159 Physical address: 10007 Value: -123
Virtual address: 23816 Physical address: 27144 Value: 0
Virtual address: 12497 Physical address: 7121 Value: 0
Virtual address: 8483 Physical address: 18467 Value: 72
Virtual address: 28609 Physical address: 18113 Value: 0
Virtual address: 16079 Physical address: 23247 Value: -77
Virtual address: 6695 Physical address: 13095 Value: -119
Virtual address: 27768 Physical address: 19576 Value: 0
Virtual address: 2430 Physical address: 16254 Value: 2
Virtual address: 14492 Physical address: 16028 Value: 0
Virtual address: 25846 Physical address: 28406 Value: 25
Virtual address: 3111 Physical address: 8487 Value: 9
Virtual address: 248 Physical address: 11256 Value: 0
Virtual address: 30281 Physical address: 27721 Value: 0
Virtual address: 14958 Physical address: 1646 Value: 14
Virtual address: 8627 Physical address: 18611 Value: 108
Virtual address: 31123 Physical address: 13459 Value: 100
Virtual address: 3057 Physical address: 21745 Value: 0
Virtual address: 2591 Physical address: 12831 Value: -121
Virtual address: 11819 Physical address: 4907 Value: -118
Virtual address: 19822 Physical address: 110 Value: 19
Virtual address: 28141 Physical address: 25837 Value: 0
Virtual address: 16843 Physical address: 29387 Value: 114
Virtual address: 11284 Physical address: 19220 Value: 0
Virtual address: 1893 Physical address: 27493 Value: 0
Virtual address: 5554 Physical address: 24242 Value: 5
Virtual address: 6731 Physical address: 13131 Value: -110
Virtual address: 17349 Physical address: 17605 Value: 0
Virtual address: 8802 Physical address: 23906 Value: 8
Virtual address: 30642 Physical address: 14002 Value: 29
Virtual address: 14420 Physical address: 15956 Value: 0
Virtual address: 28252 Physical address: 31068 Value: 0
Virtual address: 18297 Physical address: 23673 Value: 0
Virtual address: 28805 Physical address: 389 Value: 0
Virtual address: 21001 Physical address: 18697 Value: 0
Virtual address: 5750 Physical address: 20854 Value: 5
Virtual address: 30711 Physical address: 14071 Value: -3
Virtual address: 27950 Physical address: 25646 Value: 27
Virtual address: 30854 Physical address: 25478 Value: 30
Virtual address: 2509 Physical address: 16333 Value: 0
Virtual address: 17008 Physical address: 7792 Value: 0
Virtual address: 24332 Physical address: 19980 Value: 0
Virtual address: 26607 Physical address: 3823 Value: -5
Virtual address: 19254 Physical address: 12598 Value: 18
Virtual address: 11447 Physical address: 19383 Value: 45
Virtual address: 31699 Physical address: 31443 Value: -12
Virtual address: 28630 Physical address: 18134 Value: 27
Virtual address: 31324 Physical address: 24668 Value: 0
Virtual address: 9945 Physical address: 19929 Value: 0
Virtual address: 19594 Physical address: 17290 Value: 19
Virtual address: 31611 Physical address: 31355 Value: -34
Virtual address: 26177 Physical address: 8001 Value: 0
Virtual address: 4894 Physical address: 8990 Value: 4
Virtual address: 2376 Physical address: 16200 Value: 0
Virtual address: 24040 Physical address: 27368 Value: 0
Virtual address: 24072 Physical address: 776 Value: 0
Virtual address: 9647 Physical address: 1199 Value: 107
Virtual address: 15712 Physical address: 14176 Value: 0
Virtual address: 15413 Physical address: 13621 Value: 0
Virtual address: 11511 Physical address: 19447 Value: 61
Virtual address: 3497 Physical address: 3497 Value: 0
Virtual address: 8034 Physical address: 24930 Value: 7
Virtual address: 19514 Physical address: 17210 Value: 19
Virtual address: 1325 Physical address: 2605 Value: 0
Virtual address: 13164 Physical address: 10604 Value: 0
Virtual address: 7235 Physical address: 29507 Value: 16
Virtual address: 14757 Physical address: 15013 Value: 0
Virtual address: 7941 Physical address: 24837 Value: 0
Virtual address: 9293 Physical address: 10829 Value: 0
Virtual address: 4281 Physical address: 2489 Value: 0
Virtual address: 6434 Physical address: 14626 Value: 6
Virtual address: 6019 Physical address: 28547 Value: -32
Virtual address: 1791 Physical address: 1535 Value: -65
Virtual address: 26773 Physical address: 26005 Value: 0
Virtual address: 16260 Physical address: 21380 Value: 0
Virtual address: 10841 Physical address: 2137 Value: 0
Virtual address: 10916 Physical address: 2212 Value: 0
Virtual address: 23412 Physical address: 17780 Value: 0
Virtual address: 26045 Physical address: 22717 Value: 0
Virtual address: 29578 Physical address: 5770 Value: 28
Virtual address: 4762 Physical address: 2970 Value: 4
Virtual address: 25979 Physical address: 22651 Value: 94
Virtual address: 18382 Physical address: 23758 Value: 17
Virtual address: 25801 Physical address: 28361 Value: 0
Virtual address: 14790 Physical address: 15046 Value: 14
Virtual address: 22546 Physical address: 16658 Value: 22
Virtual address: 8198 Physical address: 5126 Value: 8
Virtual address: 4625 Physical address: 2833 Value: 0
Virtual address: 9884 Physical address: 19868 Value: 0
Virtual address: 659 Physical address: 28819 Value: -92
Virtual address: 20481 Physical address: 5377 Value: 0
Virtual address: 1612 Physical address: 1356 Value: 0
Virtual address: 12394 Physical address: 7018 Value: 12
Virtual address: 6638 Physical address: 14830 Value: 6
Virtual address: 25344 Physical address: 5888 Value: 0
Virtual address: 6601 Physical address: 14793 Value: 0
Virtual address: 22136 Physical address: 6776 Value: 0
Virtual address: 12694 Physical address: 8854 Value: 12
Virtual address: 6228 Physical address: 8276 Value: 0
Virtual address: 823 Physical address: 15671 Value: -51
Virtual address: 30030 Physical address: 21070 Value: 29
Virtual address: 21915 Physical address: 6555 Value: 102
Virtual address: 24523 Physical address: 20171 Value: -14
Virtual address: 2608 Physical address: 12848 Value: 0
Virtual address: 9434 Physical address: 10970 Value: 9
Virtual address: 8731 Physical address: 23835 Value: -122
Virtual address: 12482 Physical address: 7106 Value: 12
Virtual address: 11927 Physical address: 5015 Value: -91
Virtual address: 916 Physical address: 15764 Value: 0
Virtual address: 6236 Physical address: 8284 Value: 0
Virtual address: 17129 Physical address: 7913 Value: 0
Virtual address: 31349 Physical address: 24693 Value: 0
Virtual address: 12848 Physical address: 31536 Value: 0
Number of Translated Addresses = 500
Page Faults = 124
Page Fault Rate = 0.248
TLB Hits = 66
TLB Hit Rate = 0.132
